### PR TITLE
Master mdl 71574 54

### PR DESCRIPTION
**Pull request by safatshahin**
Pr for #54

### CODE REVIEW

File: question/classes/output/qbank_chooser.php [hunk 1]
```diff
@@ -51,6 +53,8 @@ class qbank_chooser extends \core\output\chooser {
      * @param context $context The relevant context.
      */
     public function __construct($real, $fake, $course, $hiddenparams, $context) {
+        debugging('Class qbank_chooser has been deprecated and moved to bank/editquestion,
```
Comment (guillogo) [inline question/classes/output/qbank_chooser.php:56]: Can you please comment this and add to a TODO in tracker: https://tracker.moodle.org/browse/MDL-71679

```
Class qbank_chooser has been deprecated and moved to bank/editquestion, please use qbank_editquestion\qbank_chooser instead.
line 57 of /question/classes/output/qbank_chooser.php: call to debugging()
line 106 of /question/classes/output/qbank_chooser.php: call to core_question\output\qbank_chooser->__construct()
line 1072 of /mod/quiz/classes/output/edit_renderer.php: call to core_question\output\qbank_chooser::get()
line 126 of /mod/quiz/classes/output/edit_renderer.php: call to mod_quiz\output\edit_renderer->question_chooser()
line 211 of /mod/quiz/edit.php: call to mod_quiz\output\edit_renderer->edit_page()
```

File: question/classes/output/qbank_chooser_item.php [hunk 2]
```diff
@@ -45,6 +47,8 @@ class qbank_chooser_item extends \core\output\chooser_item {
      * @param context $context The relevant context.
      */
     public function __construct($qtype, $context) {
+        debugging('Class qbank_chooser_item has been deprecated and moved to bank/editquestion,
```
Comment (guillogo) [inline question/classes/output/qbank_chooser_item.php:50]: Can you please comment this and add to a TODO in tracker: https://tracker.moodle.org/browse/MDL-71679

Right now I'm getting a warning when editing a mod_quiz

```
Class qbank_chooser_item has been deprecated and moved to bank/editquestion, please use qbank_editquestion\qbank_chooser_item instead.
line 51 of /question/classes/output/qbank_chooser_item.php: call to debugging()
line 61 of /question/classes/output/qbank_chooser.php: call to core_question\output\qbank_chooser_item->__construct()
line ? of unknownfile: call to core_question\output\qbank_chooser->core_question\output\{closure}()
line 60 of /question/classes/output/qbank_chooser.php: call to array_map()
line 106 of /question/classes/output/qbank_chooser.php: call to core_question\output\qbank_chooser->__construct()
line 1072 of /mod/quiz/classes/output/edit_renderer.php: call to core_question\output\qbank_chooser::get()
line 126 of /mod/quiz/classes/output/edit_renderer.php: call to mod_quiz\output\edit_renderer->question_chooser()
line 211 of /mod/quiz/edit.php: call to mod_quiz\output\edit_renderer->edit_page()
```

**Review by guillogo — CHANGES_REQUESTED** [review]
Hi @safatshahin 

Review done.
Code checker: OK
Unit test: OK
User test: OK
Code: I've added 2 comments in order to add the TODO instead the debugging in qbank_chooser and qbank_chooser_item. Also, can you validate if the disable and enable validation in the scripts should be also done as we chatted for another plugin (delequestion).

Thank you.

**Comment by safatshahin** [comment]
Hi @guillogo 
Made the changes as discussed, rebased with the latest helper class and called the method from the scripts.

**Comment by guillogo** [comment]
Hi @safatshahin approved and merged. You can continue now with ready for tracker check list. Thank you.

**Pull request merged**